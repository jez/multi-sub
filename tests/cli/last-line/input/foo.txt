
<!-- TODO(jez) Document .soft / .checked once API is stable. -->
