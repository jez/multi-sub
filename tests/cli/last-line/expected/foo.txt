
<!-- TODO(jez) Document .on_failure / .checked once API is stable. -->
